frequency-chain/frequency
# Fix token symbol testnet

# Goal
The goal of this PR is to fix an issue with the token symbols for testnet found while testing CheckMetadataHash extension.

Part of #2011 

# Discussion

- Corrected the symbols used for `node/service/src/chain_spec/frequency_paseo.rs`
- Corrected the symbols used in the `substrate_wasm_builder::WasmBuilder::init_with_defaults().enable_metadata_hash`
- Added a skipped e2e test that can be run to test transacting using the `metadata_hash`
- Required: Update cargo deny exclusions
- Adjacent: Updated the API Augment packages
- Adjacent: Updated the API Augment to have all the no-op signed extensions and remove warnings
- Adjacent: Updated the API Augment to list all the old v1 rpcs remove warnings

# Checklist
- [x] Updated js/api-augment for Custom RPC OR Runtime API?
- [x] Spec version incremented?

# How to Test

With the e2e test:
- Build Frequency using the feature `metadata-hash` (EZ mode: edit line 109 of `init.sh` to have it as one of the features)
- Set the `skip` in `e2e/signed-extensions/checkMetadataHash.test.ts` to be only
- Run `make js`
- `cd e2e`
- `npm i ../js/api-augment/dist/frequency-chain-api-augment-0.0.0.tgz`
- `npm run test:serial`

1. See that the test passes
2. See that there are no warnings about signed extensions
3. See that there are no warnings about rpcs

Note: You will see an warning about `PORTABLEREGISTRY: Unable to determine runtime Call type, cannot inspect sp_runtime::generic::unchecked_extrinsic::UncheckedExtrinsic` However that appears to be due to an issue with the latest version of Polkadotjs/api and not related to these changes nor does it impact the running of the api.
